src/test/resources/org/mapfish/print/config/sample_config_yaml/configPremiumGoogle.yaml

```diff
@@ -49,5 +49,4 @@ layouts:
         - !map
           spacingAfter: 30
           width: 440
-          height: 483
-      footer: *commonFooter
```

Comment:
Same as above, Why we removed the footer here?

Reply:
Same as above answer